docs/how-to/index.md

```diff
@@ -37,3 +49,15 @@ Use named "identities" to allow additional users to access the API.
 
 Manage identities <manage-identities>
 ```
+
+
+## API
```

Comment:
Can the API stuff be removed from this PR? It's in https://github.com/canonical/pebble/pull/545/files

Reply:
Will do; I added this because previously we linked this how-to to that doc.